doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -62,7 +62,10 @@ You also need the following:
             .\JLink_Windows_V794i_x86_64.exe -InstUSBDriver=1
 
       #. Follow the on-screen instructions.
-      #. After installing, add the J-Link executable to the system path on Linux and MacOS, or to the environment variables on Windows, to run it from anywhere on the system.
+      #. After installing, ensure the J-Link executable can be run from anywhere on your system:
+
+        * For Linux and MacOS, add it to the system path.
+        * For Windows, add it to the environment variables.
```

Comment:
```suggestion
         * For Linux and MacOS, add it to the system path.
         * For Windows, add it to the environment variables.
```
Also, are these instructions in the correct place? Shouldn't they be with all the other installation instructions? The note with instructions really breaks up the list of requirements:
![image](https://github.com/user-attachments/assets/fbc58f2a-623b-483a-ac48-810005e22b4e)

Reply:
Something we incorporated during one of the releases was creating a separate subsection in Software chapter outlining the instructions that have to be completed before moving forward - if the driver still doesn't get installed automatically, this could be a solution.

Reply:
This was needed here as installing jlink without the usb driver would prevent the user from being able to flash correctly the dk on windows. It was a workaround to issues arised during previous ux testings. I agree with your concern, but as the toolchain installation will change in 3.0, and this has been tested and is working, I would not move this a day before rc-1

Reply:
Alright, I can see that, even if it isn't optimal.